tutorials/cookbooks/running_production.md

```diff
@@ -0,0 +1,308 @@
+
+[Step 3: Secure and finish your setup. Using a Reverse Proxy, domain name and HTTPS](#Step-3-Secure-and-finish-your-setup-Using-a-Reverse-Proxy-domain-name-and-HTTPS)
++ [3.1. Creating a Reverse Proxy with Nginx](#31-Creating-a-Reverse-Proxy-with-Nginx)
++ [3.2. Activating SSL (HTTPS) on your MeiliSearch](#32-Activating-SSL-HTTPS-on-your-MeiliSearch)
+
+[Conclusion](#Conclusion)
+  
+## Get your MeiliSearch ready for production
+  
+For this tutorial, we will be using a Debian 10 server, running on DigitalOcean. You can easily try it on your own, with plans starting at $5/month. And if you want some credits to start running your MeiliSearch and are not already registered on DigitalOcean, you can get $100 for free using [this referral link](https://m.do.co/c/7c67bd97e101).  
+
+
+
+## Prerequisites
+
+- An up-to-date server that runs Debian 10
+- An ssh keypair to connect to that machine
+
+> *TIPS:* learn how to connect via SSH to your [DigitalOcean droplet](https://www.digitalocean.com/docs/droplets/how-to/connect-with-ssh/) or any [Linux or windows server](https://phoenixnap.com/kb/ssh-to-connect-to-remote-server-linux-or-windows)
+
+## Step 1: Install MeiliSearch
+
+Installing and running MeiliSearch is an easy and straightforward process. In order to keep this tutorial as simple as possible, we are going to use a MeiliSearch installation script that will take care of the process for you. It will get a copy of a MeiliSearch binary in your machine and let you use it inmediatley.
+
+Once we are logged in into our machine via SSH, we only need to be sure that our system and its dependencies are up-to-date and install MeiliSearch.
+```bash
+# Update references to the dependencies on Debian
+$ apt update
+# Install curl, that we need to install MeiliSearch in the next step
+$ apt install curl -y
+# Install MeiliSearch latest version from the script
+$ curl -L https://install.meilisearch.com | sh
+```
+
+> The diferent options to achieve a MeiliSearch install are detailed in our **[Installation Docs](https://docs.meilisearch.com/guides/advanced_guides/installation.html#download-and-launch)**
+
+> It is important to know that there are different ways to get MeiliSearch running on your machine. As an open source project, you can always compile the latest stable release of MeiliSearch from the source code to ensure the binary uses your achitecture in the best possible way.  
+
+You can always check the latest MeiliSearch stable version, and get MeiliSearch for the Operating System of your choice, by visiting the following link:  
+
+**[Latest MeiliSearch Stable Version](https://github.com/meilisearch/MeiliSearch/releases/latest)**  
+
+MeiliSearch is finally installed and ready to use. If you want to make it accessible from everywhere in your system, you should move this binary into your system binaries folder, with the following command:  
+
+```bash
+# Move the MeiliSearch binary to your system binaries
```

Comment:
Already explained in the line above.
```suggestion
```

Reply:
Developers normally read the examples, and not necessarily the article. That's why I think it is important to comment the code and commands even if its repetitive, it is easy to ignore the comment. What do you think?

Reply:
Yes, agreed! I cancel my suggestion or you can resolve it.